pubspec.yaml

```diff
@@ -16,6 +16,5 @@ dev_dependencies:
 flutter:
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
-  flutter: ">=1.12.0"
-
+  sdk: ">=2.13.0 <3.0.0"
```

Comment:
Could these constraints be lowered to earliest version with null safety support?